src/main/java/com/refinedmods/refinedstorage/apiimpl/autocrafting/task/v6/node/ProcessingNode.java

```diff
@@ -132,6 +132,8 @@ public void update(INetwork network, int ticks, NodeList nodes, IStorageDisk<Ite
                         if (canInsertFullAmount) {
                             canInsertFullAmount = IoUtil.insertIntoInventory(container.getConnectedFluidInventory(), extractedFluids.getStacks(), Action.SIMULATE);
                         }
+                    } else {
+                        break;
                     }
 
                     if (hasAllRequirements && !canInsertFullAmount) {
```

Comment:
Btw now `hasAllRequirements` is always true here (and can be removed) and the next if can be removed entirely (not removed but replaced with its content), since both `hasAllRequirements` and `canInsertFullAmount` are always true at that point)